src/main/java/IPChat.java

```diff
@@ -0,0 +1,41 @@
+import java.util.Scanner;
+public class IPChat {
+    public static void main(String[] args) {
+
+        // Level 3: Mark as done
+        System.out.println("Hello I'm IPChat, What can I do for you");
+        System.out.println("------------------------------------------");
```

Comment:
Code quality: Avoid magic strings

Consider using a constant `LINE_DIVIDER` to denote a line divider, which can then be reused and will improve readability.